### PR TITLE
KDE Connect

### PR DESCRIPTION
**Pull request by Max-F-Helm**
App-Profile for the KDE Connect Daemon.

Settings:
- override the "Block Incoming Connections" to allow connections

### CODE REVIEW

**Comment by Raphty** [comment]
@Max-F-Helm thank you for the pr, the import of your profile does not work.

it seems your id is still an old one. it should have been migrated, but it seems to have had an issue.

it will work if you remove the two lines above.

also if you get in touch that we can look into what went wrong with the id migration that would be great.

**Comment by Max-F-Helm** [comment]
@Raphty
with 
> it will work if you remove the two lines above

you mean these lines?

```yaml
type: profile
id: 4e386bc1-3826-4e6c-bba9-422b34128f9d
```


Regarding getting in touch, what would be your preferred way of communication?

File: app-profiles/services-kdeconnectd.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+type: profile
+id: 4e386bc1-3826-4e6c-bba9-422b34128f9d
```
[the remark this answers is not in the document]
Comment (Raphty) [reply app-profiles/services-kdeconnectd.yml]: line 2

File: app-profiles/services-kdeconnectd.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    - type: path
+      operation: equals
+      value: /usr/libexec/kdeconnectd
+config:
+    filter:
+        blockInbound: 6
+lastEdited: 2022-11-17T10:59:01+01:00
+created: 2022-11-17T10:30:55+01:00
+
+# jess-checksum: Zwk9P3YRS1iDY9YFMyvuyoRymcFqYPAjbP1XM3ijJEs6nc
```
[the remark this answers is not in the document]
Comment (Raphty) [reply app-profiles/services-kdeconnectd.yml]: line 17

**Comment by Raphty** [comment]
no I meant line 2 and 17, i wanted to make the comment in line so it is easier to see (but I get it is not as clear as I wanted it to be 😅 )

thank you for the Contributions!

**Comment by Max-F-Helm** [comment]
PR updated, pleas check if it works now.

**Review by Raphty — APPROVED** [review]

**Review by dhaavi — APPROVED** [review]

**Comment by Raphty** [comment]
@dhaavi just made me aware that I should also have recommended an additional change, to block internet for the app so you don't allow connections from the internet to the kde connect app

image

**Comment by Raphty** [comment]
@Max-F-Helm we can make the change or you can 😄

**Pull request merged**